aws-opentelemetry-distro/src/amazon/opentelemetry/distro/patches/_botocore_patches.py

```diff
@@ -58,9 +59,9 @@ def patch_extract_attributes(self, attributes: _AttributeMapT):
         queue_name = self._call_context.params.get("QueueName")
         queue_url = self._call_context.params.get("QueueUrl")
         if queue_name:
-            attributes["aws.sqs.queue_name"] = queue_name
+            attributes[AWS_QUEUE_NAME] = queue_name
```

Comment:
Thanks for changing to constants. I don't know why we didn't do this in the first place.